### PR TITLE
Prepare for release 2.0.8

### PR DESCRIPTION
**Pull request by laurentS**
This is a bugfix release:
- Added support for Russian language
- Updated some Czech translations
- Fixed a bug that prevented some users from completing the tutorial
- Minor dependency upgrades

### CODE REVIEW

File: src/shared/views/Mapper/CardBody.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ class _CardBody extends React.PureComponent<Props, State> {
         this.flatlist = null;
         this.scrollEnabled = !props.tutorial;
         this.tasksPerScreen = undefined;
```
Comment (Hagellach37) [inline src/shared/views/Mapper/CardBody.js:80]: what does this line say? why do you set it to `undefined` here. 

don't we expect for the the "normal" project type, that tasks per screen is always 6?

File: src/shared/views/Mapper/CardBody.js [hunk 2]
```diff
@@ -360,10 +360,20 @@ class _CardBody extends React.PureComponent<Props, State> {
...
         let result = 0;
         if (this.tasksPerScreen) {
-            result = this.tasksPerScreen[screenNumber].reduce(
-                (sum, task) => sum + task.referenceAnswer,
-                0,
-            );
+            console.log('tps', this.tasksPerScreen.length, screenNumber);
+            // $FlowFixMe
+            if (this.tasksPerScreen[screenNumber]) {
+                result = this.tasksPerScreen[screenNumber].reduce(
```
Comment (Hagellach37) [inline src/shared/views/Mapper/CardBody.js:366]: I'm a bit confused here. I assume that `result` refers to the number of tasks expected for this screen, as it is defined by the sum of the reference answers.

**Review by Hagellach37 — APPROVED** [review]
Hey,
I can follow the things in general. And I think it makes sense to do it in this way. But still we need to learn more about the cause of the issue.

I guess that it goes somewhere in the direction of firebase data being loaded so quick enough, but still this would need further backing.

File: src/shared/views/Mapper/CardBody.js (hunk 2 above)
Comment (laurentS) [reply src/shared/views/Mapper/CardBody.js:366]: Yes, it's the total number of taps expected on the screen, if the user does a "perfect sequence". I had put that in the comment at the top of the function, let me know if I should clarify

File: src/shared/views/Mapper/CardBody.js (hunk 1 above)
Comment (laurentS) [reply src/shared/views/Mapper/CardBody.js:80]: I set it to `undefined` as a way to indicate that data hasn't loaded from the backend yet. Once it is loaded, `generateTasks()` fills it in with an array of `number of screens` items, each of which is 6 tasks long.

**Pull request merged**